README.md

```diff
@@ -54,14 +53,15 @@ You may override the following variables to customise your certificate:
 
 * `ORGANISATION_DOMAIN` The domain name of your organisation, e.g. `example.com`
 * `ORGANISATION_UNIT` The (optional) name of your organisation unit, e.g. `Database`
-* `OUTPUT_DIR` The directory in which the `ca` output directory will be created
+* `OUTPUT_DIR` The directory in which the `ca` output directory will be created - note: must be absolute
 * `COUNTRY_CODE` The two-letter country code in which your orrganisation resides, e.g. `US`
 * `STATE_NAME` The name of the state in which your orrganisation resides, e.g. `WA`
 * `LOCALITY_NAME` The name of the city in which your orrganisation resides, e.g. `Perth`
 * `EMAIL_ADDRESS` The email address that should be used to contact you with questions about your certificate
 * `COMMON_NAME` Allows you to customise the commmon name of the certificate
 * `CLIENT_KEY_PASSWORD` You can set this instead of entering the passphrase every time your key is used
 * `INTERMEDIATE_KEY_PASSWORD` You can set this instead of entering the passphrase every time your intermediate key is used
+* `CERTIFICATE_TYPE` You can set this to `usr_cert` or `server_cert` subject to your needs - defaults to `usr_cert`
```

Comment:
I would much prefer that the terminology here was `client_cert` since I think that most certificate consumers are APIs not users.